docs/samples/collect_multi_agent_samples.sh

```
@@ -0,0 +1,170 @@
+  if [ $exit_code -eq 124 ]; then
+    echo "  ⏰ 超时 (${TIMEOUT_SEC}s)"
+  else
+    echo "  退出码: $exit_code | stdout: ${stdout_size}bytes | stderr: ${stderr_size}bytes"
+  fi
+
+  # 写入输出文件
+  {
+    echo "# $display_name 多 Agent 协作真实输出样本"
+    echo "# 采集时间: $(date -u +'%Y-%m-%dT%H:%M:%SZ')"
+    echo "# 任务: 刘会计派生张三丰(8+8) + 李雷(10*10) 并汇总"
+    echo "# 执行命令: $exec_path ${cmd_parts[@]:1} <prompt>"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)